pubspec.yaml

```diff
@@ -24,6 +24,7 @@ dependencies:
   flutter_loggy: ^2.0.2
   flutter_loggy_dio: ^3.1.0
   get_it: ^7.6.9
+  icons_plus: ^5.0.0
```

Comment:
If we are using this package, maybe you can try to remove the line `uses-material-design: true` below?

We remove the `flutter create` comments from this file but it was saying it is especially declared to allow usage of material icons.